spring-rabbit/src/main/java/org/springframework/amqp/rabbit/listener/BlockingQueueConsumer.java

```diff
@@ -97,8 +97,6 @@
 
 	private InternalConsumer consumer;
 
-	private final AtomicBoolean cancelled = new AtomicBoolean(false);
-
 	private final AtomicBoolean cancelReceived = new AtomicBoolean(false);
```

Comment:
Since this is now overloaded to mean either a `cancel` or `cancelOk` has been received, we should probably rename it to something like `consumerCancelled` instead and add javadoc `cancel or canceOk received from broker`.